### PR TITLE
Added tick marks support for horizontal slider bar. 

### PR DESCRIPTION
**Pull request by tony0918**
- Added tick marks support for horizontal slider bar.
- Changed demo file to present a demo usage for the slider with tick marks. 
- Fixed jquery include error when jquery lib updated to 2.x.

### CODE REVIEW

File: src/slider.js [hunk 1]
```diff
@@ -135,6 +135,31 @@ angular.module('ui.slider', []).value('uiSliderConfig',{}).directive('uiSlider',
...
+    var properties = ['max', 'step', 'tick'];
+    angular.forEach(properties, function(property) {
+        if (angular.isDefined(attrs[property])) {
+            options[property] = attrs[property];
+        }
+    });
+    if (angular.isDefined(options['tick']) && angular.isDefined(options['step'])) {
+        var total = parseInt(parseInt(options['max'])/parseInt(options['step']));
+        for (var i = total; i >= 0; i--) {
+            console.log(i);
```
Comment (PowerKiKi) [inline src/slider.js]: that should not be there...

Comment (tony0918) [inline src/slider.js]: Do you mean that we need to use anonymous function? I've followed the code style here: https://github.com/johnpapa/angularjs-styleguide#style-y070

Comment (PowerKiKi) [inline src/slider.js]: Sorry I just mean the ``console.log`` shouldn't be there obviously...

Comment (tony0918) [inline src/slider.js]: OK. I've added another commit that fixed this. And also moved this function into compile section. Please have a review. Thanks.

**Comment by PowerKiKi** [comment]
thx!

**Pull request merged**